airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/orchestrator/ReplicationJobOrchestrator.java

```diff
@@ -158,6 +161,12 @@ public Optional<String> runJob() throws Exception {
         sourceLauncherConfig.getDockerImage());
 
     log.info("Setting up replication worker...");
+    final UUID workspaceId = syncInput.getWorkspaceId();
```

Comment:
Any possibility that `syncInput.getWorkspaceId();` returns null?

Reply:
Yes indeed, will check for it in the next revision here.